getting_started/setup_linux/setup_ktest_environment.rst

```diff
@@ -44,27 +43,60 @@ Copy your existing known-to-work kernels/initrd
 
 .. code-block:: bash
 
-   cp /boot/vmlinuz-4.13.0-16-generic /boot/vmlinuz-test
-   cp /boot/initrd.img-4.13.0-16-generic /boot/initrd.img-test
+   sudo cp /boot/vmlinuz-`uname -r` /boot/vmlinuz-test
 
-Change the extensions as needed to create an initial grub entry
-for a test kernel. You will never override the default
-Ubuntu/Debian stuff, so you will always have the ability to boot a
+   # On Ubuntu:
+   sudo cp /boot/initrd.img-`uname -r` /boot/initrd.img-test
+   
+   # On Fedora:
+   sudo cp /boot/initramfs-`uname -r`.img /boot/initramfs-test.img
+   sudo grubby --add-kernel /boot/vmlinuz-test --title=test
+
+
+You will never override the default
+distro kernel, so you will always have the ability to boot a
 working kernel if your changes fail to boot.
 
-4. Edit grub default
---------------------
+4. Edit grub settings
+---------------------
+
+This only needs to be run on Ubuntu and Debian, Fedora has the proper settings by default.
 
 .. code-block:: bash
 
    # Use your text editor of choice.
    sudo emacs /etc/default/grub
+   # Change GRUB_DEFAULT=[n] to GRUB_DEFAULT=saved
+   # Then add GRUB_DISABLE_SUBMENU=y to the end and save,
+   # submenus confuse ktest.
    sudo update-grub
 
-Add ``GRUB_DISABLE_SUBMENU=y`` to the end and save.
-Sub-menus confuse ktest.
+5. Set the default kernel
```

Comment:
This section should come after "getting familiar with grub-reboot". The previous one probably too.

Reply:
I think it makes sense to leave as-is. The default kernel and grub config only need to be run once, whereas the grub-reboot stuff is somewhat continual. The current order is a bit more chronological in that way.

Reply:
Fat-fingering any of these commands could cause an unbootable system, regardless of familiarity with grub-reboot. That's not something we can avoid.

The commands it tells you to run before the "get familiar" section don't use grub-reboot at all, they just set configuration required to use grub-reboot. The cheatsheet doesn't work until they are run. For example, it shows how to use `grub-reboot` to boot to the test kernel. If that gets run before the steps in the previous two sections, they're computer could default to the new kernel or not boot the new kernel at all, leading the user to think they did something wrong. I'll let you or @plbossart have the final say on this, but I definitely think the order should be left as-is.